### ISSUE TITLE
Bar stays on top of everything fullscreen (i3) 

### ISSUE THREAD

**Issue by soyuka** [geommer/yabar#128]
Hi,

First of all, big thanks for building this, this is one of the best bar I've tested so far!

## Description

I've an issue on i3 with `yabar`, when I enter full screen mode (mod+f or vlc full screen), the bar stays on top of other windows, an my content get's truncated.

`yabar` gets started from i3 with `exec --no-startup-id yabar`, and I've two bars (bottom and top). So that i3 gets the top/bottom margin right, I had to keep two `bar{}` declarations in the i3 configuration.
`yabar` is executed at the end of the i3 configuration.

I tried without success to use `xdo` to tweak x-windows orders.

## Versions

i3-gaps (master)
yabar (dev_workspaces) (works lovely <3 thanks @NBonaparte )
archlinux

## Related

https://github.com/geommer/yabar/issues/120
https://github.com/geommer/yabar/issues/71 ? I've read this one carefully but I'm not really sure that my issue is related as I've no issues with blocks.
https://github.com/geommer/yabar/issues/116 might be related if I have to close yabar when entering full screen and start it again on exit. Should I? Any tips on how this should be handled?

I've C abilities and can manually install yabar.

Thanks!

**Comment by soyuka** [comment]
#129 related (duplicate?)

**Comment by soyuka** [comment]
I've solved this by using [a script to toggle yabar](https://gist.github.com/soyuka/c86dd69cfc85fc12a7cc98d703090d1f) and a change in `i3` config:

```
bindsym $mod+f exec "bash ~/.config/i3/yabar.sh || 1 && i3 fullscreen toggle"
```

**Comment by NBonaparte** [comment]
I'll see if this can be fixed internally. I might have a solution for this, but I haven't tested it yet (I'm unfortunately not an i3 user).

**Comment by NBonaparte** [comment]
I think I fixed this with 63f0bb2, could you check? Sorry for breaking it in the first place :man_facepalming:

**Comment by soyuka** [comment]
Oh wow okay nice! I'll try this ASAP! What wm are you using?

**Comment by soyuka** [comment]
fixed thanks <3 that should also fix #130 because I can now remove the i3 fake bars :D.
